build/common.gypi

```diff
@@ -3,6 +3,7 @@
     'tizen%': 0,
     'tizen_mobile%': 0,
     'enable_murphy%': 0,
+    'enable_webcl%': 1,
```

Comment:
It's sad that the other variables here have no explanation; can you add some comments about this one, as is the case for the variables in Chromium's `build/common.gypi`?